### ISSUE TITLE
Error when running sample code from TL;DR

### ISSUE THREAD

**Issue by cparker4486** [EsOsO/Logging#7]
Hello,

Trying to use the TL;DR from Logging.psd1 results in some errors. First the code being run:

```
Set-LoggingDefaultLevel -Level 'WARNING'
Add-LoggingTarget -Name Console
Add-LoggingTarget -Name File -Configuration @{Path = 'C:\my\path\example_%{+%Y%m%d}.log'}

$Level = 'DEBUG', 'INFO', 'WARNING', 'ERROR'
foreach ($i in 1..100) {
    Write-Log -Level ($Level | Get-Random) ('Message n.{0}' -f $i)
    Start-Sleep -Milliseconds (Get-Random -Min 100 -Max 1000)
}

Wait-Logging        # See Note
```

And the errors:

```
New-Object : Exception calling ".ctor" with "1" argument(s): "Cannot process argument because the value of argument 
"validValues" is out of range. Change argument "validValues" to  a value that is within range.
Parameter name: validValues
Actual value was System.String[]."
At C:\Users\<user>\OneDrive - <tenant>\WindowsPowerShell\Modules\Logging\Logging.psm1:307 char:33
+ ... Attribute = New-Object System.Management.Automation.ValidateSetAttrib ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [New-Object], MethodInvocationException
    + FullyQualifiedErrorId : ConstructorInvokedThrowException,Microsoft.PowerShell.Commands.NewObjectCommand
 
New-Object : Exception calling ".ctor" with "1" argument(s): "Cannot process argument because the value of argument 
"validValues" is out of range. Change argument "validValues" to  a value that is within range.
Parameter name: validValues
Actual value was System.String[]."
At C:\Users\<user>\OneDrive - <tenant>\WindowsPowerShell\Modules\Logging\Logging.psm1:307 char:33
+ ... Attribute = New-Object System.Management.Automation.ValidateSetAttrib ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [New-Object], MethodInvocationException
    + FullyQualifiedErrorId : ConstructorInvokedThrowException,Microsoft.PowerShell.Commands.NewObjectCommand
 
Cannot index into a null array.
At C:\Users\<user>\OneDrive - <tenant>\WindowsPowerShell\Modules\Logging\Logging.psm1:290 char:13
+         if ($TargetConf.Configuration[$Conf] -and $Configuration[$Con ...
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : NullArray
```

**Comment by EsOsO** [comment]
Could you please post the output of `$PSVersionTable`?

**Comment by cparker4486** [comment]
```
Name                           Value
----                           -----
PSVersion                      5.1.14393.206
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.14393.206
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

**Comment by EsOsO** [comment]
Hi, @cparker4486, I have the same version as you. Could you try to `Import-Module Logging` before executing the example code?

**Comment by cparker4486** [comment]
@EsOsO The module imported without a problem but I got the same result. I'll try it on another computer and report back.

**Comment by cparker4486** [comment]
@EsOsO I've tried on two other computers. One with Windows 10 LTSB and one Server 2012 R2. Same result in both places. Here's $PSVersionTable from the latter computer.

```
Name                           Value
----                           -----
PSVersion                      5.0.10586.117
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.10586.117
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1

```

**Comment by EsOsO** [comment]
It seems like you are missing the target folder inside the module one:
```
Logging
    |
    +---- targets
```

Have you git cloned the repo or installed the module from powershelgallery?

**Comment by cparker4486** [comment]
Well this is embarrassing... but, yes, it's all user error. I downloaded the zip and and only copied the .psd1 and .psm1 into the module folder.

On the bright side, this shows that there may be room for improvement in the README. It doesn't say anything about installing so I went about "installing" it myself. I didn't consider looking for it in the gallery because that's not something I'm accustomed to doing.

Sorry to have wasted your time on this!
